### ISSUE TITLE
To fetch the demographic information of users engaged in a post

### ISSUE THREAD

**Issue by rowedenny** [kevinzg/facebook-scraper#158]
Thanks for the work.

Following collecting the number of likes/comments/shares, I would like to collect the demographic information of the users who engaged in a post.  More specifically, given a post with 98 likes, I would like to investigate the age and gender distribution of them. 

So I am wondering if it is possible to make it, given I can only get access to the profile information publicly available.

**Comment by kevinzg** [comment]
I don't think Facebook freely publish that information.

**Comment by neon-ninja** [comment]
This PR (https://github.com/kevinzg/facebook-scraper/pull/201) in conjunction with this PR (https://github.com/kevinzg/facebook-scraper/pull/188, already merged) could make this possible

**Comment by neon-ninja** [comment]
As of https://github.com/kevinzg/facebook-scraper/pull/194#issuecomment-815325534 you can now extract users reacting to a post. Note that Like is just one of 7 possible reaction types.

**Comment by ambrel5** [comment]
Hi there, I keep getting an error when running the above code:

![image](https://user-images.githubusercontent.com/64226828/116816774-a3041d80-ab63-11eb-8b89-77de36c2a0f2.png)

What am I doing wrong?

**Comment by neon-ninja** [comment]
yeah, if you're not logged in / passing cookies, commenter_url will always be None
